include/Budget.h

```diff
@@ -1,27 +1,30 @@
+//Created By Ahmed Hamdy Kotp
+
 #ifndef Budget_H
 #define Budget_H
 #include <string>
-#include <vector>
 #include <map>
 
-using namespace std;
-
 
-class Budget
-{
+using namespace std;
+class Budget :  Person {
 private :
 	map<string, double> categories;
-    
-	
+    double total_budget;
 public :
+    //Constructor
     Budget();
-    double total_budget;
-	void add_category(const string& category_name, double amount);
-    double getBalance(const string& category_name) const;
-    void spend(const string& category_name, double amount);
+
+    //setters
```

Comment:
those three functions are methods not setters. Setters usually start with 'set'. In this example, setTotalBudget only

Reply:
don't delete those functions, just move them to methods section.